python-modules/cis_profile_retrieval_service/cis_profile_retrieval_service/v2_api.py

```diff
@@ -61,21 +61,21 @@ def scope_to_mozilla_data_classification(scopes):
     classifications = []
     if "classification:mozilla_confidential" in scopes:
         logger.debug("Mozilla confidential data classification in scope.")
-        classifications.append(MozillaDataClassification.MOZILLA_CONFIDENTIAL)
+        classifications.extend(MozillaDataClassification.MOZILLA_CONFIDENTIAL)
```

Comment:
I guess this is what we wanted to do here. `MozillaDataClassification` are lists.